gmsm/games-ios
# Create Project

**Goal**
Create `TopGames` project

**How was it done?**
I created a project with the Single View Application model.

**How to test?**
When you download the repository, you will see the `TopGames` folder inside the `games-ios` repository. This can be confirmed by the terminal with the command `ls -a`

**Evidence (optional)**
